### PR TITLE
mermaid-cli 8.8.4 (new formula)

### PR DESCRIPTION
**Pull request by BastianZim**
mermaid-cli 8.8.4 (new formula)

Add formula

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by BastianZim** [comment]
@carlocab This would be the new PR.

Thanks again for your help and sorry for the extra work.

File: Formula/mermaid-cli.rb
Comment (carlocab) [inline Formula/mermaid-cli.rb:4]:
```suggestion
  desc "Command-line interface (CLI) for mermaid"
```

File: Formula/mermaid-cli.rb
Comment (carlocab) [inline Formula/mermaid-cli.rb:40]:
```suggestion
    assert_predicate testpath/"out.svg", :exist?
```

**Review by carlocab — COMMENTED** [review]
Thanks again, @BastianZim.

**Comment by BastianZim** [comment]
@carlocab Seems like chromium is not yet available for ARM: https://github.com/Homebrew/homebrew-core/pull/67804/checks?check_run_id=1614088150#step:7:563

Is there anything I can do or should I just report that to the maintainers?

**Comment by carlocab** [comment]
Yes, I noticed... https://chromium.googlesource.com/chromium/src.git/+/master/docs/mac_arm64.md

Let's just wait and see if others have feedback on this before we decide what to do next.

**Comment by carlocab** [comment]
@fxcoudert do you have an opinion on this? It seems like a popular piece of software: https://www.npmjs.com/package/@mermaid-js/mermaid-cli

**Comment by BastianZim** [comment]
Just to add: The problem comes from the package [puppeteer](https://github.com/puppeteer/puppeteer) which interacts with Chromium.

From what I can see there, no official support is yet provided, the only way is running it using Rosetta. https://github.com/puppeteer/puppeteer/issues/6622

**Comment by chrmoritz** [comment]
I think the questions is here more if we want to ship the bottle with a downloaded third-party compiled chromium binary here or do we want to just set the [`PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` env var](https://github.com/puppeteer/puppeteer/blob/main/install.js#L61) and maybe tell the user in a caveat how to point it to a locally installed Chrom{e,ium] with `PUPPETEER_EXECUTABLE_PATH`.

**Comment by fxcoudert** [comment]
@chrmoritz let's try to minimise the number of chromium builds we distribute as part of other things

**Comment by carlocab** [comment]
Sounds like pupeteer downloads and installs a pre-compiled Chromium binary, though. Does that disqualify this from homebrew-core?

**Comment by chrmoritz** [comment]
I think that wouldn't be a problem, since puppeteer will also download it's precompiled chromium binary during runtime, if `PUPPETEER_EXECUTABLE_PATH` doesn't point to a valid Chrom{e,ium] executable. ` PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` just skips the download during the installation, if I read the code at https://github.com/puppeteer/puppeteer/blob/f63a123ecef86693e6457b07437a96f108f3e3c5/src/node/Launcher.ts#L606 correctly.

File: Formula/mermaid-cli.rb
Comment (dawidd6) [inline Formula/mermaid-cli.rb:38]:
```suggestion
    system bin/"mmdc", "-p", "puppeteer-config.json", "-i", "#{testpath}/test.mmd", "-o", "#{testpath}/out.svg"
```

**Review by chenrui333 — APPROVED** [review]

**Comment by BastianZim** [comment]
Hi everybody, just wanted to check if there is anything else I can do or if we're just waiting for further feedback concerning ARM?

**Comment by chenrui333** [comment]
Nope. Good to 🚢 !! 

Thanks @BastianZim your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**
